### PR TITLE
PT-1345 Blip Aqua microscanner vulnerabilities

### PR DESCRIPTION
**Pull request by yvon-dblg**
- Use nexus
- Replace salinity with chai
- Bump deps
- Update travis config

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,14 +1,19 @@
...
+before_install:
+  - npm install -g npm@latest
 install:
   - npm install
 
 script:
   - npm test
-
-matrix:
-  allow_failures:
```
Comment (franck-fourel) [inline .travis.yml:13]: I think we should keep the allow-failures on the node version that are not production mandatory, ie lts/*

File: package.json [hunk 2]
```diff
@@ -4,23 +4,24 @@
...
   "devDependencies": {
-    "grunt": "1.0.4",
+    "chai": "4.2.0",
+    "grunt": "1.1.0",
     "grunt-cli": "1.3.2",
     "grunt-contrib-jshint": "2.1.0",
     "grunt-mocha-test": "0.13.3",
-    "mocha": "6.2.2",
-    "salinity": "0.0.8"
+    "mocha": "8.0.1"
```
Comment (franck-fourel) [inline package.json:25]: general comment: why do we make those upgrade in this PR ?
do we need them to address the PR objective ?

**Review by franck-fourel — CHANGES_REQUESTED** [review]
questioning more than requesting changes:
I'm still wondering why we need this update apart from using our private npm registry. 
can you specify the changelog as well ?

File: package.json (hunk 2 above)
Comment (yvon-dblg) [reply package.json:25]: They embark dependencies with vulnerabilities.
which then comes to blip, replacing sometimes fixed packages versions.
npm seems to use the oldest version on conflict (at least I got a problem with one dep).

File: .travis.yml [hunk 3]
```diff
@@ -1,14 +1,23 @@
...
+  - lts/*
+
+matrix:
+  allow_failures:
+    - node_js: lts/*
 
+cache: npm
+
+before_install:
+  - npm install -g npm@latest
```
Comment (franck-fourel) [inline .travis.yml]:
```suggestion
  - npm install -g npm@6.14.5
```

**Review by franck-fourel — CHANGES_REQUESTED** [review]
small change before accepting it

**Review by franck-fourel — APPROVED** [review]
LGTM

**Pull request merged**